packages/ui-library/src/components/select/index.ts

```diff
@@ -82,6 +82,10 @@ export class BlrSelect extends LitElementCustom {
     }
   };
 
+  protected firstUpdated(_changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>): void {
+    this.handleSlotChange();
```

Comment:
Now it gets called twice, right? Once from the <slot @slotchange={..}> and once from firstUpdated.

Reply:
Correct. It's only a hotfix until then